### PR TITLE
Demon Prison F1

### PR DESCRIPTION
**Pull request by vtange**
**@imcgames:**

How should the word, Kupole/Kyupol/Cupole/Cupol be treated? Right now we have it all as "**Kupole**", but if you have any preferences (I personally prefer Cupol since it looks similar to "Cupid"), do tell me :)

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -6505,36 +6505,36 @@ QUEST_LV_0200_20150406_006508	{memo X}You successfully destroyed the magic circl
...
+QUEST_LV_0200_20150406_006530	It seems that Kupole Daiva made an important decision. Talk to Kupole Daiva.
+QUEST_LV_0200_20150406_006531	Kupole Daiva told you that now is the time to show him your real strength. Defeat the demons that hid at Huradeti Crossroads and collect the teeth of the demons that are their symbol.
+QUEST_LV_0200_20150406_006532	You've collected the teeth of the demons. Take them to Kupole Daiva.
+QUEST_LV_0200_20150406_006533	Kupole Sigita seems to have a request to you who is exhausted after using all her power blocking the crossover space. Talk to Kupole Sigita.
+QUEST_LV_0200_20150406_006534	Kupole Sigita told you that the demons shouldn't be gathered together for the future. Defeat the demons that are gathered at the Banaga Monitor District.
+QUEST_LV_0200_20150406_006535	You've defeated all demons near Banaga Monitor District. Talk to Kupole Sigita.
+QUEST_LV_0200_20150406_006536	Kupole Sigita is worried about Dionys, who is suffering. Talk to Kupole Sigita.
+QUEST_LV_0200_20150406_006537	To recover Dionys, Kupole Sigita asked you to collect the empty spirits from the demons that are gathered at Pasaru isolated area.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:6533]: Dionys -> Dionysus

**Comment by Sunaries** [comment]
Good to merge.

@imcgames Can you tell us the meaning behind 큐폴 (Kupole) - We were also wondering if that is the right spelling for it.

**Comment by imcgames** [comment]
큐폴 is Low or Junior rank Goddess in TOS. 
Actually, it's secret meaning, so we want users to know this by playing.

**Comment by Sunaries** [comment]
> 큐폴 is Low or Junior rank Goddess in TOS. 
> Actually, it's secret meaning, so we want users to know this by playing.

@imcgames I see, thanks. But how is it exactly spelled? Because we are having a discussion on Discord on this matter.
We are wondering if it spelled: Kupole, Cupol, Cupole, Kupol etc. 

_(I am inclined to believe the spelling is not Kyupol, because thats the Korean transliteration.)_

**Comment by imcgames** [comment]
What do you think of Kupol?

**Comment by imcgames** [comment]
:+1:

**Pull request merged**